lms/static/js/dashboard/views/course_view.js

```diff
@@ -0,0 +1,161 @@
+;(function (define, accessibleModal) {
+    'use strict';
+
+    define(['jquery', 'backbone', 'underscore', 'underscore.string',
+            'text!templates/dashboard/course.underscore'],
+        function ($, Backbone, _, _s, courseTemplate) {
+
+            if (_.isUndefined(_s)) {
+                _s = _.str;
+            }
+
+            var CourseView = Backbone.View.extend({
+                el: '<li class="course-item" />',
+                events: {
+                    'click .action-more': 'toggleCourseActionsDropDown',
+                    'click .action-unenroll': 'unEnroll',
+                    'click .action-email-settings': 'emailSettings',
+                    'click #upgrade-to-verified': 'upgradeToVerified',
+                    'click #block-course-msg a[rel="leanModal"]': 'unRegisterBlockCourse'
+                },
+
+                initialize: function (options) {
+                    this.template = _.template(courseTemplate);
```

Comment:
The can be defined as a parameter for the view instead of in the initialize function, e.g.
template: _.template(courseTemplate),